pom.xml

```diff
@@ -148,13 +148,39 @@
                 <version>3.6.0</version>
                 <configuration>
                     <filters>
+                        <filter>
```

Comment:
These filters were modified to clear up warnings about overlapping resources when assembling the shaded JAR.

https://ge.solutions-team.gradle.com/s/occfwoiar7gn4/console-log?page=1#L38-L49

This ensures the only LICENSE, NOTICE, and MANIFEST.MF files included in the shaded JAR are those from this project.